refinery/tool_manager/models.py

```diff
@@ -0,0 +1,136 @@
+from django.db import models
+from django_extensions.db.fields import UUIDField
+
+
+class Parameter(models.Model):
+    """
+    A Parameter is a representation of Galaxy/Visualization/Other tool
+    parameters that will potentially be exposed and configurable upon a tool's
+    configuration/launching step.
+    """
+
+    INTEGER = "INTEGER"
+    STRING = "STRING"
+    BOOLEAN = "BOOLEAN"
+    FLOAT = "FLOAT"
+    GENOME_BUILD = "GENOME_BUILD"
+    ATTRIBUTE = "ATTRIBUTE"
+    FILE = "FILE"
+
+    VALUE_TYPES = (
+        (INTEGER, "int"),
+        (STRING, "str"),
+        (BOOLEAN, "bool"),
+        (FLOAT, "float"),
+        (GENOME_BUILD, "genome build"),
+        (ATTRIBUTE, "attribute"),
+        (FILE, "file")
+    )
+    uuid = UUIDField(unique=True, auto=True)
+    name = models.TextField(max_length=100, blank=False,
+                            null=False)
+    description = models.TextField(max_length=500, blank=False,
+                                   null=False)
+    is_editable = models.BooleanField(default=False)
+    value_type = models.CharField(choices=VALUE_TYPES, max_length=25,
+                                  blank=False, null=False)
+    default_value = models.TextField(max_length=100, blank=False, null=False)
+    # ID of Galaxy tool that the parameter belongs to
+    galaxy_tool_id = models.TextField(max_length=300, blank=False, null=False)
```

Comment:
I think that should work. Btw, no need for `null=False` - it's a default https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.Field.null